docs/Developers/Newcomers_Guide.rst

```diff
@@ -424,6 +424,12 @@ The ``Pull Request`` will automatically update with the newest changes.
 Now you should `tell us about your experience <https://coala.io/newform>`_ and
 go for `a low issue <https://coala.io/low>`__ - they are really rewarding!
 
+
+.. note::
+
+    Do not delete the fork/s subsequent to Pull Request for review or
```

Comment:
The issue states to add a note saying not to delete fork after PR. Just made it `fork or forks`.